Base/QTGUI/qSlicerWebPythonProxy.h

```diff
@@ -52,7 +68,7 @@ public slots:
   /// running python code from web pages.
   ///
   /// \sa qSlicerWebWidget::initializeWebEngineProfile
-  QString evalPython(const QString &python);
+  QString evalPython(const QString &python, int mode = FileInput);
```

Comment:
I ended up using `int` as the type because I didn't find a way to make it work with:

```cpp
QString evalPython(const QString &python, EvalPythonMode mode = FileInput);
```